docs/ref/contrib/postgres/fields.txt

```diff
@@ -916,3 +916,44 @@ types.
     .. attribute:: range_type
 
         The psycopg2 range type to use.
+
+Expression support for range boundaries
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. class:: RangeBoundary(lower=LOWER_INC, upper=UPPER_EXC)
+
+For use together with
+:class:`django.contrib.postgres.constraints.ExclusionConstraint`, a simple
+utility class is provided that provides an easier way to write the range
+boundaries. The limits are described in the `documentation`_. Internally,
+``RangeBoundary`` uses :class:`~django.db.models.Value`.
```

Comment:
Does it? Can't see where.

Reply:
Yes, I initially experimented with `Value`, and forgot to update this bit. Thanks :+1: